common/src/IO/ObjSerializer.h

```diff
@@ -103,6 +103,7 @@ namespace TrenchBroom {
 
             Path m_objPath;
             Path m_mtlPath;
+			Path m_gamePath;
```

Comment:
I think you are using tabs for indentation? Please make sure you indent with spaces only.